src/components/Modal/style.css

```diff
@@ -48,9 +57,8 @@
   background-color: #f3f3f4;
 
   color: #585E5F;
-  font-size: 16*@unit;
   font-weight: 400;
-  line-height: 24*@unit;
+  line-height: 1.5;
```

Comment:
so they need to change line height to ems as well or not?

Reply:
We could do, but I just generally changed any `line-height` reference to be a ratio of the font-size, rather than a specific size. It makes life a bit easier (especially when working with `em`s, and dynamically changing font-sizes) and is often what's really called for anyway